src/browser/modules/Stream/Stream.tsx

```diff
@@ -50,9 +50,13 @@ import {
 } from 'shared/modules/connections/connectionsDuck'
 import { getScrollToTop } from 'shared/modules/settings/settingsDuck'
 import DbsFrame from './Auth/DbsFrame'
-import EditFrame from './EditFrame'
+import { StyledFrame } from '../Frame/styled'
+import FrameTitlebar from '../Frame/FrameTitlebar'
+import { dim } from 'browser-styles/constants'
+import styled from 'styled-components'
+import ExportButton, { ExportItem } from './ExportButtons'
 
-const trans = {
+const nameToFrame: Record<string, React.ComponentType<any>> = {
```

Comment:
Love that you've given this a name that's actually descriptive!

Reply:
And it also ryhmes